__tests__/app.test.js

```diff
@@ -238,6 +238,17 @@ describe("GET /api/articles/:article_id/comments", () => {
         });
       });
   });
+  test("200:Responds with an empty array when article ID has 0 comments", () => {
+    const article_id = 2;
+    return request(app)
+      .get(`/api/articles/${article_id}/comments`)
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(comments).toBeInstanceOf(Array);
+        expect(comments.length).toBe(0);
```

Comment:
This is great - we need the same thing to be added to the test above where it's responding with an array that does have comments, please!